patch/0001-SPARK-LLAP-RANGER-Integration_2.2.0.patch

```diff
@@ -0,0 +1,345 @@
+diff --git a/resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/ConfigurableCredentialManager.scala b/resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/ConfigurableCredentialManager.scala
```

Comment:
Like the this. Your name should be there.
https://github.com/hortonworks-spark/spark-llap/blob/master/patch/0001-SPARK-LLAP-RANGER-Integration_2.1.0.patch#L1-L18